manifests/charts/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -203,6 +208,10 @@ spec:
         secret:
           secretName: cacerts
           optional: true
+      - name: remote-kube-config
```

Comment:
perhaps `central-kube-config`?

Reply:
`istio-kubeconfig` might be better as this is a way to define kubeconfig for istiod and is not limited to central mode.

Reply:
+1 on `istio-kubeconfig` - fits different scenarios.